src/dotnet/commands/dotnet-cache/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.MSBuild;
+
+namespace Microsoft.DotNet.Tools.Cache
+{
+    public partial class CacheCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false);
+            app.Name = "dotnet cache";
+            app.FullName = LocalizableStrings.AppFullName;
+            app.Description = LocalizableStrings.AppDescription;
+            app.AllowArgumentSeparator = true;
+            app.ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText;
+            app.HelpOption("-h|--help");
+
+            CommandArgument projectArgument = app.Argument($"<{LocalizableStrings.ProjectArgument}>",
+                LocalizableStrings.ProjectArgDescription,
+                true);
+
+            CommandOption frameworkOption = app.Option(
+                $"-f|--framework <{LocalizableStrings.FrameworkOption}>", LocalizableStrings.FrameworkOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption runtimeOption = app.Option(
+                $"-r|--runtime <{LocalizableStrings.RuntimeOption}>", LocalizableStrings.RuntimeOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption outputOption = app.Option(
+                $"-o|--output <{LocalizableStrings.OutputOption}>", LocalizableStrings.OutputOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption fxOption = app.Option(
+                $"--fx-version <{LocalizableStrings.FxVersionOption}>", LocalizableStrings.FxVersionOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption workingDir = app.Option(
+               $"--working-dir <{LocalizableStrings.WorkingDirOption}>", LocalizableStrings.WorkingDirOptionDescription,
```

Comment:
This command though operates on a project file, is not operating in context of a folder. think it is like more of dotnet install. I am providing this option for the sake of scriptability, for example it could be PWD